boxyhq/jackson
# Sbom 1

## What does this PR do?

Github Action for creating and saving SBOM reports in SPDX & CycloneDx format for the following
1. Jackson npm package codebase
2. Next Js application codebase
3. Next Js application docker image

Fixes #36 

## Type of change

DevSecOps

## How should this be tested?

Just make a commit and push to sbom-1 branch to see the workflow in the actions 
Test to check if correct files are being created at correct locations

## Checklist:
